spec/rubocop/cop/rspec/sort_tags_spec.rb

```diff
@@ -0,0 +1,279 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::SortTags do
+  it 'does not register an offense when using only symbol tags ' \
+     'in alphabetical order' do
+    expect_no_offenses(<<~RUBY)
+      RSpec.describe 'Something', :a, :b do
```

Comment:
Do you mind adding a couple of weird examples, e.g. multi-line metadata:
```ruby
RSpec.describe 'Something',
               :foo, :bar,
               baz: 'goo' do
```

and some weird values:
```ruby
RSpec.describe 'Something', foo: ->(x) { bar(x) }
```
Would the cop work correctly?

Reply:
In the multi-line case, what would be your expected offense and expected correction?

Reply:
There should be no difference. I'm just slightly concerned that multi-line might not work as expected, and if it's a major pain to make it work, it would make sense to just ignore it.